### ISSUE TITLE
Why is unserialize_noclass only available for PHP 8+ ?

### ISSUE THREAD

**Issue by milansimek** [jvoisin/snuffleupagus#501]
According to [the docs](https://snuffleupagus.readthedocs.io/config.html#unserialize-noclass) `unserialize_noclass` is available for PHP 8+.   

I'm wondering whether there's a specific reason for this since the `allowed_classes` option is available in PHP 7 too according to the [PHP docs](https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.filtered-unserialize)? 

Thanks for clarifiying!

**Comment by jvoisin** [comment]
I think that the main reason was that I was too lazy to implement it for PHP7 as well, but [it shouldn't be hard to do it](https://github.com/jvoisin/snuffleupagus/blob/master/src/sp_unserialize.c#L99-L111) :)

**Comment by jvoisin** [comment]
Good catch, feel free to send another pull-request :P

**Comment by milansimek** [comment]
Will do! Just need to finish the current project I'm working on first, although that's almost done :)  

And I'll have to invest some time into setting up a dev environment for this purpose since I've never worked on PHP module development before.  

Do you recommend to use CLion or another ide? Begin able to use breakpoints for debugging would make things a lot easier for sure. 

(My experience writing C is limited to some smaller embedded projects only)

**Comment by jvoisin** [comment]
I'm a terrible person and I use vim for everything. As for the setup, `git clone` + `make` should be enough :)

**Comment by milansimek** [comment]
Yeah same with me, I use vim pretty much the entire day. And when working in an IDE I always use vim keybindings. 

Can't live without it that's for sure :)  

When working on a large new project, in this case the PHP source code, some breakpoints and a debugger help me get familiar much faster though. I'll figure it out, thanks!
